web/src/views/FindSpeaker.vue

```diff
@@ -25,23 +26,29 @@
 import SpeakerCard from '@/components/SpeakerCard.vue';
 import Search from '@/components/Search.vue';
 
-import db from '../plugins/airtable';
-
 export default {
   components: {
     Search,
     SpeakerCard,
   },
   data: () => ({
     speakers: [],
+    prefectures: [],
     error: null,
   }),
   mounted() {
+    this.$getLocations(this.setPrefectures, this.setError);
     this.getSpeakers();
   },
   methods: {
+    setPrefectures(records) {
+      this.prefectures = records;
+    },
+    setError(err) {
+      this.error = err;
```

Comment:
Should we at least log the error in these callbacks?

Reply:
probably a good idea, let me add that